generators/app/templates/prettierrc

```diff
@@ -1,11 +1 @@
-{
-  "printWidth": 80,
-  "tabWidth": 2,
-  "useTabs": false,
-  "semi": false,
-  "singleQuote": true,
-  "trailingComma": "all",
-  "bracketSpacing": false,
-  "jsxBracketSameLine": false,
-  "proseWrap": "always"
-}
+module.exports = require('kcd-scripts/prettier')
```

Comment:
I think this filename needs to be changed

Reply:
It's the same name as the main `prettierrc.js` file, but without the extension.

Do yo want me to add the extension in the template file?

Reply:
Yeah, let's use the extension. In fact, let's change this to prettier.config.js

Reply:
You always use `.prettierrc.js` in your projects so that's why I named it like that